pyproject.toml

```diff
@@ -6,35 +6,34 @@ authors = ["Team Microdata"]
 license = "Apache"
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = "^3.12"
 fastapi = "^0.109.1"
-pyarrow = "^14.0.1"
+pyarrow = "^15.0.0"
 uvicorn = "^0.15.0"
-google-cloud-storage = "^1.42.3"
-pandas = "^1.3.3"
+google-cloud-storage = "^2.14.0"
```

Comment:
Is this dependency still used in the code?

Reply:
No, I cannot find anything in the code.
Only in README (under configuration) - which also need an update?

There are currently two configuration profiles: LOCAL and GCS. You need to set environment variable CONFIG_PROFILE to one of the values. In case of GCS you need to have GOOGLE_APPLICATION_CREDENTIALS in environment pointing to JSON key file. 

The configuration files are ```data_service/config/.env.gcs``` and ```data_service/config/.env.local_file```.

Reply:
👍  Yes, we should also update README. We can remove the whole section `## Configuration` and also remove the line in docker run command `--env CONFIG_PROFILE=LOCAL \`. And `CONFIG_PROFILE=LOCAL` line from .env file in root directory of this project.